src/platform_impl/android/mod.rs

```diff
@@ -768,7 +767,7 @@ impl Window {
     }
 
     pub fn outer_size(&self) -> PhysicalSize<u32> {
-        MonitorHandle::new(self.app.clone()).size()
+        screen_size(&self.app)
```

Comment:
Maybe we should rename this helper to `surface_size()` or inline it entirely?

Reply:
Couldn't inline it because its called in two places.
But renamed!